### PR TITLE
Do not assume a header when a custom title node is given

### PR DESCRIPTION
**Pull request by brunobasto**
## Description

Uses the provided header node instead of allowing customizing the variant via `titleVariant`.

## Checklist

- [ ] This PR has good automated test coverage
- [ ] The storybook for the component has been updated

## Steps to Test

1. Pull down PR.
1. `npm run dev`.
1. Open http://localhost:6006/?path=/story/wizard--custom-title
1. Verify that the given `title` prop of each step is not wrapped by a `h4`.

### CODE REVIEW

File: src/system/Wizard/WizardStep.js [hunk 1]
```diff
@@ -56,19 +48,26 @@ const WizardStep = ( {
...
+				<Heading
+					variant="h4"
+					sx={ {
+						mb: 0,
+						display: 'flex',
+						alignItems: 'center',
+						color: color,
+					} }
+				>
+					<MdCheckCircle sx={ { mr: 2 } } />
```
Comment (djalmaaraujo) [inline src/system/Wizard/WizardStep.js]: I guess we don't need this being announced. Can you add `aria-hidden="true"`

File: src/system/Wizard/WizardStep.js [hunk 2]
```diff
@@ -56,19 +48,26 @@ const WizardStep = ( {
...
+					{ title }
+				</Heading>
+			) : (
+				<Flex sx={ { alignItems: 'center', color } }>
+					<MdCheckCircle sx={ { mr: 2 } } />
```
Comment (djalmaaraujo) [inline src/system/Wizard/WizardStep.js]: I guess we don't need this being announced. Can you add `aria-hidden="true"`

File: src/system/Wizard/WizardStepHorizontal.js [hunk 3]
```diff
@@ -9,24 +9,31 @@ import PropTypes from 'prop-types';
...
 				mb: 0,
 				display: 'flex',
 				alignItems: 'center',
-				color: active ? 'heading' : 'muted',
+				color,
 			} }
 			data-step={ order }
 			data-active={ active || undefined }
 		>
 			<MdCheckCircle sx={ { mr: 2 } } />
```
Comment (djalmaaraujo) [inline src/system/Wizard/WizardStepHorizontal.js]: I guess we don't need this being announced. Can you add `aria-hidden="true"`

File: src/system/Wizard/WizardStepHorizontal.js [hunk 4]
```diff
@@ -9,24 +9,31 @@ import PropTypes from 'prop-types';
...
 			{ title }
 		</Heading>
+	) : (
+		<Flex sx={ { alignItems: 'center', color } }>
+			<MdCheckCircle sx={ { mr: 2 } } />
```
Comment (djalmaaraujo) [inline src/system/Wizard/WizardStepHorizontal.js]: I guess we don't need this being announced. Can you add `aria-hidden="true"`

**Review by djalmaaraujo — CHANGES_REQUESTED** [review]
Small thing.

**Comment by brunobasto** [comment]
Thank you. Fixed.

**Review by djalmaaraujo — APPROVED** [review]
Thank you @brunobasto! 🥳

**Pull request merged**